spec/index.bs

```diff
@@ -3898,6 +3945,52 @@ dictionary GPURenderPassDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPURenderPassDescriptor>
+    : <dfn>colorAttachments</dfn>
+    ::
+        The set of {{GPURenderPassColorAttachmentDescriptor}} values in this sequence defines which
+        color attachments will be output to when executing this render pass.
+
+    : <dfn>depthStencilAttachment</dfn>
+    ::
+        The {{GPURenderPassDepthStencilAttachmentDescriptor}} value that defines the depth/stencil
+        attachment that will be output to and tested against when executing this render pass.
+
+    : <dfn>occlusionQuerySet</dfn>
+    ::
+        Issue: Describe this dictionary member
+</dl>
+
+<div class=validusage dfn-for=GPURenderPassDescriptor>
```

Comment:
hm, these should be algorithms, but we can't actually trivially make them algorithms easily with `<div algorithm class=validusage ...>` because they'll just end up with the algorithm name "Valid Usage". We'll figure this out later.